crates/aptos/src/config/mod.rs

```diff
@@ -179,6 +179,23 @@ fn global_folder() -> CliTypedResult<PathBuf> {
     }
 }
 
+fn find_workspace_config(starting_path: PathBuf) -> CliTypedResult<PathBuf> {
+    let mut current_path = starting_path.clone();
+    loop {
+        let cand = current_path.join(CONFIG_FOLDER);
+        if cand.is_dir() {
+            break Ok(cand);
```

Comment:
this is an interesting notation... why not return instead of break?

Reply:
This is weird too, I'd just do return as well

Reply:
This is c&p from Tim's code of the package system, I also did not know this was possible, but I actually like it. Lets keep it here in memory of Tim. There is nothing wrong with it, it might be more idiomatic Rust then the return...